ci/compute-build-args.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -euo pipefail
+
+ARGS=""
+case "${IMAGE_REPO}" in
+  ci)
+    ARGS="CUDA_VER=${CUDA_VER}
+LINUX_VER=${LINUX_VER}
+PYTHON_VER=${PYTHON_VER}"
+    ;;
+  *)
+    MANYLINUX_VER="manylinux_2_17"
+    if [[
+        "${LINUX_VER}" == "ubuntu18.04" ||
+        "${LINUX_VER}" == "ubuntu20.04"
+    ]]; then
+        MANYLINUX_VER="manylinux_2_31"
+    fi
+    ARGS="CUDA_VER=${CUDA_VER}
+LINUX_VER=${LINUX_VER}
+PYTHON_VER=${PYTHON_VER}
+CPU_ARCH=${ARCH}
+REAL_ARCH=$(arch)
+MANYLINUX_VER=${MANYLINUX_VER}"
+    ;;
+esac
+
+echo "ARGS<<EOF" >> "$GITHUB_OUTPUT"
+echo "$ARGS" >> "$GITHUB_OUTPUT"
+echo "EOF" >> "$GITHUB_OUTPUT"
```

Comment:
```suggestion
MANYLINUX_VER="manylinux_2_17"
if [[
  "${LINUX_VER}" == "ubuntu18.04" ||
  "${LINUX_VER}" == "ubuntu20.04"
]]; then
  MANYLINUX_VER="manylinux_2_31"
fi

ARGS=(
  # common args
  "CUDA_VER=${CUDA_VER}"
  "LINUX_VER=${LINUX_VER}"
  "PYTHON_VER=${PYTHON_VER}"
  # wheel args
  "CPU_ARCH=${ARCH}"
  "REAL_ARCH=$(arch)"
  "MANYLINUX_VER=${MANYLINUX_VER}"
)

cat <<EOF > "${GITHUB_OUTPUT:-/dev/stdout}"
ARGS<<EOT
$(printf "%s\n" "${ARGS[@]}")
EOT
EOF
```

I don't think `docker build` penalizes us for providing too many build arguments (I think it just issues a warning).

Therefore, we might be able to flatten and simplify this a bit by doing something like this suggestion.

Also, using `"${GITHUB_OUTPUT:-/dev/stdout}"` ensures that these scripts can be tested locally.